CentralityiOS/CentralityiOS/AppDelegate.m

```diff
@@ -0,0 +1,54 @@
+//
+//  AppDelegate.m
+//  CentralityiOS
+//
+//  Created by Samuel Osa-Agbontaen on 7/8/22.
+//
+
+#import "AppDelegate.h"
+#import "Parse/Parse.h"
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    ParseClientConfiguration *config = [ParseClientConfiguration  configurationWithBlock:^(id<ParseMutableClientConfiguration> configuration) {
+
+        NSString *path = [[NSBundle mainBundle] pathForResource: @"Keys" ofType: @"plist"];
+        NSDictionary *dict = [NSDictionary dictionaryWithContentsOfFile: path];
+
+        NSString *applicationId = [dict objectForKey: @"applicationId"];
+        NSString *clientKey = [dict objectForKey: @"clientKey"];
+        
+        configuration.applicationId = [NSString stringWithFormat:@"%@", applicationId];
+        configuration.clientKey = [NSString stringWithFormat: @"%@", clientKey];
+        configuration.server = @"https://parseapi.back4app.com";
+    }];
+
+    [Parse initializeWithConfiguration:config];
+    return YES;
+}
+
+
+#pragma mark - UISceneSession lifecycle
+
+
+- (UISceneConfiguration *)application:(UIApplication *)application configurationForConnectingSceneSession:(UISceneSession *)connectingSceneSession options:(UISceneConnectionOptions *)options {
+    // Called when a new scene session is being created.
+    // Use this method to select a configuration to create the new scene with.
+    return [[UISceneConfiguration alloc] initWithName:@"Default Configuration" sessionRole:connectingSceneSession.role];
+}
+
+
+- (void)application:(UIApplication *)application didDiscardSceneSessions:(NSSet<UISceneSession *> *)sceneSessions {
+    // Called when the user discards a scene session.
+    // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
+    // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
+}
+
+
```

Comment:
Looks like this is an empty method? Please remove if this is not needed here!

Reply:
This is a part of the `UIApplicationDelegate` protocol, so I'm pretty sure it needs to be here.